src/Files.App/Views/Layouts/BaseLayoutPage.cs

```diff
@@ -1176,11 +1194,12 @@ private void RefreshItem(SelectorItem container, object item, bool inRecycleQueu
 			}
 		}
 
-		protected static void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
+		protected internal void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
```

Comment:
Is internal needed?
```suggestion
		protected void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
```

Reply:
> Is internal needed?

I'm not sure, I just saw that the method below it added `internal`
``` C#
protected internal void FileListItem_PointerEntered(object sender, PointerRoutedEventArgs e)
```